### PR TITLE
Adds missing options to drop down lists

### PR DESCRIPTION
**Pull request by samuel-rosa**

### CODE REVIEW

**Comment by samuel-rosa** [comment]
The schema address was adjusted after validation using https://github.com/datacite/bolognese

File: index.html [hunk 1]
```diff
@@ -135,17 +148,152 @@
...
+		optionValues["nameIdentifierScheme"] = ["ORCID", "ISNI", "ROR", "GRID", "GND", "Lattes"];
+		optionValues["affiliationIdentifierScheme"] = [
+			"ROR",
+			"GRID",
+			"ISNI",
+			"Wikidata",
+			"Crossref Funder ID",
+			"Other"
+		]
+		optionValues["subjectScheme"] = ["DDC", "GND", "Wikidata"];
```
Comment (Zack-83) [inline index.html:296]:
```suggestion
		optionValues["nameIdentifierScheme"] = ["ORCID", "ISNI", "ROR", "GRID", "GND", "Lattes"];
		optionValues["affiliationIdentifierScheme"] = ["ROR", "GRID", "ISNI", "Wikidata", "Crossref Funder ID", "Other" ];
		optionValues["subjectScheme"] = ["DDC", "GND", "Wikidata"];

```

File: index.html [hunk 2]
```diff
@@ -135,17 +148,152 @@
...
+		optionValues["funderIdentifierType"] = [
+			"Crossref Funder ID",
+			"GRID",
+			"ISNI",
+			"ROR",
+			"EU",
+			"DFG",
+			"FWF",
+			"Other"
+		];
```
Comment (Zack-83) [inline index.html:285]:
```suggestion
		optionValues["identifierType"] = ["DOI"]
		optionValues["descriptionType"] = ["Abstract", "Methods", "SeriesInformation", "TableOfContents", "TechnicalInfo", "Other"];
		optionValues["relatedIdentifierType"] = ["ARK", "arXiv", "bibcode", "DOI", "EAN13", "EISSN", "Handle", "IGSN", "ISBN", "ISSN", "ISTC", "LISSN", "LSID", "PMID", "PURL", "UPC", "URL", "URN", "w3id"];
		optionValues["relationType"] = ["IsCitedBy", "Cites", "IsSupplementTo", "IsSupplementedBy", "IsContinuedBy", "Continues", "IsDescribedBy", "Describes", "HasMetadata", "IsMetadataFor", "HasVersion", "IsVersionOf", "IsNewVersionOf", "IsPreviousVersionOf", "IsPartOf", "HasPart", "IsReferencedBy", "References", "IsDocumentedBy", "Documents", "IsCompiledBy", "Compiles", "IsVariantFormOf", "IsOriginalFormOf", "IsIdenticalTo", "IsReviewedBy", "Reviews", "IsDerivedFrom", "IsSourceOf", "IsRequiredBy", "Requires", "IsObsoletedBy", "Obsoletes"];
		optionValues["resourceTypeGeneral"] = ["Audiovisual", "Collection", "DataPaper", "Dataset", "Event", "Image", "InteractiveResource", "Model", "PhysicalObject", "Service", "Software", "Sound", "Text", "Workflow", "Other"];
		optionValues["dateType"] = ["Accepted", "Available", "Copyrighted", "Collected", "Created", "Issued", "Submitted", "Updated", "Valid", "Withdrawn", "Other"];
		optionValues["contributorType"] = ["ContactPerson", "DataCollector", "DataCurator", "DataManager", "Distributor", "Editor", "HostingInstitution", "Producer", "ProjectLeader", "ProjectManager", "ProjectMember", "RegistrationAgency", "RegistrationAuthority", "RelatedPerson", "Researcher", "ResearchGroup", "RightsHolder", "Sponsor", "Supervisor", "WorkPackageLeader", "Other"];
		optionValues["titleType"] = [// "MainTitle",
			"AlternativeTitle", "Subtitle","TranslatedTitle","Other"];
		optionValues["funderIdentifierType"] = ["Crossref Funder ID", "GRID", "ISNI", "ROR", "Other"];

```

**Comment by MSpenger** [comment]
Changes updated and added in Pull Request https://github.com/UB-LMU/datacite-metadata-generator/pull/11

**Pull request closed without merging**